network/proto/aggregate_data.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+package aggregate_data;
+
+message BigInt {
+    bytes bytes = 1;
+}
+
+message PreCalculatedAggregates {
+    string timestamp = 1; // Timestamp for the given day/hour that this aggregated data is associated with
+    BigInt blocks = 2; // Only used for daily tracking and not hourly
```

Comment:
IIRC, `bytes` don't get displayed properly in the console (displayed as bytes and not human readable). That was the reason I used string instead. Any chance SF fixed this?

Reply:
Yeh good catch - will put this back to i64. I believe it should be readable